scalarflow/charts/data-population/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Default values for data-population.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+dataPopulation:
+  image:
+    repository: ghcr.io/scalar-labs/scalarflow-data-population
+    pullPolicy: IfNotPresent
+    tag: 1.0.0
+  # This will clean the data population job after the job is finished (either completed or failed) after a set number of seconds. "0" means disable this feature (the job is not cleaned).
+  ttlSecondsAfterFinished: 60
+  # Arguments for the scalarflow data population
+  args: []
+  
+  scalardb:
+    config: scalardb.properties
+    configData: ""
+    # In the case the properties file is not used, then the configuration can be done via bellow configs
+    namespace: scalarflow
```

Comment:
It seems that this value is not used in any resources.
Can we remove this value?
Or, do we have to apply this value to any resources?

Reply:
It is missed from the job spec. I updated it. Thank you for pointing it out